geode-core/src/main/java/org/apache/geode/internal/cache/GemFireCacheImpl.java

```diff
@@ -1086,6 +1088,35 @@ public static GemFireCacheImpl getForPdx(String reason) {
     clientMetadataService = clientMetadataServiceFactory.apply(this);
   }
 
+  @Override
+  public void lockDiskStore(String diskStoreName) {
+    CountDownLatch countDownLatch = diskStoreLatches.get(diskStoreName);
```

Comment:
Is get() required, when using putIfAbsent()? putIfAbsent will return either null or old value.

Reply:
Good point! I will make the change per your suggestion.